packages/eslint-plugin-formatjs/tests/blocklist-elements.test.ts

```diff
@@ -76,7 +76,7 @@ ruleTester.run(name, rule, {
   ],
 })
 
-vueRuleTester.run('vue/blocklist-elements', rule, {
+vueRuleTester.run(`vue-${name}`, rule, {
```

Comment:
The name with a slash broke with the new `RuleTester`, since it's manufacturing a plugin with a `@ruletester` prefix on the fly, and creates a config with a single rule matching `@ruletester/${name}`.  When the name has another prefix on top of that, it messes up the plugin -> rule name matching.